text/0000-create-stackify-repo.md

```diff
@@ -0,0 +1,141 @@
+# Meta
+[meta]: #meta
+- Name: Create stackify repo
+- Start Date: 2020-11-06
+- Authors: @martyspiewak @mdelillo @dumez-k
+- RFC Pull Request: (leave blank)
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: N/A
+
+# Summary
+[summary]: #summary
+Create a new repo called “stackify” which can be used to turn OCI images into CNB compliant stack images.
+
+# Definitions
+[definitions]: #definitions
+N/A
+
+# Motivation
+[motivation]: #motivation
+Currently, for users to extend the published stack images, or turn their own images into stack images, they have to create Dockerfiles which take the base images, apply their changes, and then set the CNB metadata. It can be difficult to generate and set all of the relevant metadata, especially the mixins.
+
+Stackify will be a tool that they can use to perform all of these manipulations to their base image without Dockerfiles.
+
+
+# What it is
+[what-it-is]: #what-it-is
+Stackify will be a tool that will be useful to users who would like to customize the published stack images or turn their own base images into CNB compliant stack images.
```

Comment:
This sounds like stackify is something new that is being created, rather than an existing tool/repo being donated to the project. Are we trying to keep that tone throughout the RFC, or is there any objection to explicitly linking to the current project?

Reply:
We have a first implementation of this that we've worked on but not to all of the specifications in this RFC. It is also currently in a private repo so we couldn't link to it.